ui/app/pages/settings/contact-list-tab/edit-contact/edit-contact.component.js

```diff
@@ -26,9 +26,9 @@ export default class EditContact extends PureComponent {
   }
 
   state = {
-    newName: '',
-    newAddress: '',
-    newMemo: '',
+    newName: this.props.name,
+    newAddress: this.props.address,
+    newMemo: this.props.memo,
```

Comment:
It would probably be a good idea to add default props for these as well, so that they fallback to being empty string. Empty string is checked for down on line 109, and it may be the anticipated default elsewhere as well.

Reply:
Is this what you mean https://github.com/MetaMask/metamask-extension/pull/7326/commits/9d793015d39b2483ac190b53ada93e1b43d136a8?

Reply:
Yep, that should do it!